packages/frontend/src/components/RefreshDbtButton/index.tsx

```diff
@@ -154,11 +168,14 @@ const RefreshDbtButton = () => {
             <Button
                 size="xs"
                 variant="default"
-                leftIcon={<MantineIcon icon={IconRefresh} />}
+                leftIcon={leftIcon ?? <MantineIcon icon={IconRefresh} />}
                 loading={isLoading}
-                onClick={onClick}
+                onClick={handleRefresh}
+                sx={buttonStyles}
             >
-                {!isLoading ? 'Refresh dbt' : 'Refreshing dbt'}
+                {!isLoading
+                    ? defaultTextOverride ?? 'Refresh dbt'
+                    : refreshingTextOverride ?? 'Refreshing dbt'}
```

Comment:
```suggestion
                    ? defaultTextOverride ?? 'Refresh catalog'
                    : refreshingTextOverride ?? 'Refreshing catalog'}
```

I think it would be better (cc @PriPatel in case you have a different opinion)

Reply:
This cannot be overridden here, this button is reused across the app
Needs to be done in `packages/frontend/src/features/metricsCatalog/components/MetricsCatalogPanel.tsx`

Reply:
🤦 that's exactly what I meant sorry